lua/pixelui/elements/cl_sidebar.lua

```diff
@@ -36,18 +36,18 @@ end
 function PANEL:Paint(w, h)
     local textCol = PIXEL.Colors.SecondaryText
     local backgroundCol = PIXEL.Colors.Transparent
-    local backgroundThingCol = ARSENIC.Colors.Transparent
+    local backgroundStyleCol = PIXEL.Colors.Transparent
```

Comment:
Variable name is still misleading. You're drawing a line on the left of the button, that's not a background.

Reply:
AEFGY UASER ITY*F SRFG

Reply:
i agree